alkira/resource_alkira_service_pan.go

```diff
@@ -208,6 +208,11 @@ func resourceAlkiraServicePan() *schema.Resource {
 								},
 							},
 						},
+						"allow_traffic": {
+							Description: "Allow traffic on the PAN instance." + " Default is `true`",
```

Comment:
I think `enable_traffic` will be a better field name?  We usually use `enable_` or `disable_` prefix.